Whiskey/Tasks/Invoke-WhiskeyExec.ps1

```diff
@@ -40,21 +40,28 @@ function Invoke-WhiskeyExec
         return
     }
 
-    if ( -not [IO.Path]::IsPathRooted($path) )
-    {
-        $path = Join-Path -Path $TaskContext.BuildRoot -ChildPath $path
-    }
+    $path = & {
+                    if( [IO.Path]::IsPathRooted($path) )
+                    {
+                        $path
+                    }
+                    else 
+                    {
+                        Join-Path -Path (Get-Location).Path -ChildPath $path
+                        Join-Path -Path $TaskContext.BuildRoot -ChildPath $path
+                    }
+            } |
+            Where-Object { Test-Path -path $_ -PathType Leaf } |
+            Select-Object -First 1 |
+            Resolve-Path |
+            Select-Object -ExpandProperty 'ProviderPath'

```

Comment:
We need to call `Select-Object -First 1` before this line because if the user specified a wildcard value for their `WorkingDirectory` property, or worse the path contains literal wildcard values, that resolves to multiple path items then `Resolve-Path` will return more than one object and the task will fail to execute the command.

Or we use the `-LiteralPath` parameter with `Resolve-Path`

Reply:
I added a check if `Resolve-Path` returns multiple paths. I can see users wanting to use wildcards to find an executable. We just want to make sure it resolves to *one* executable.